oneandone/src/test/java/org/apache/jclouds/oneandone/rest/internal/BaseOneAndOneLiveTest.java

```diff
@@ -102,6 +105,16 @@ protected Server createServer(String serverName) {
               .powerOn(Boolean.TRUE).build());
    }
 
+   protected Server updateServerStatus(Server server) {
```

Comment:
This is just to get the latest state of the Server from the API its more like getServer nothing is being blocked except that sometimes users need the latest Server state, what do you think?